frontend/server/configs.ts

```diff
@@ -79,13 +79,13 @@ export function loadConfigs(argv: string[], env: ProcessEnv): UIConfigs {
     /** Envoy service will listen to this port */
     METADATA_ENVOY_SERVICE_SERVICE_PORT = '9090',
     /** Is Argo log archive enabled? */
-    ARGO_ARCHIVE_LOGS = 'false',
+    ARGO_ARCHIVE_LOGS = 'true',
```

Comment:
We should keep the default as 'false' to be on the same page as KFP. Then we can document it at the [admin doc](https://github.com/kubeflow/kfp-tekton/blob/master/kfp-admin-guide.md) to show then how to enable this feature.

Since KFP also didn't expose this feature in configmap, we have use the below commands to enable this.
```
kubectl set env -n kubeflow deploy/ml-pipeline-ui ARGO_ARCHIVE_LOGS=true
```